postprocessing/pp-ext-report.h

```diff
@@ -0,0 +1,26 @@
+#ifndef JANUS_GATEWAY_PP_EXT_REPORT_H
+#define JANUS_GATEWAY_PP_EXT_REPORT_H
+
+#include <inttypes.h>
+
+#include "pp-rtp.h"
+
+typedef struct janus_pp_extension_report_rotation {
+	int rotation;
+	double timestamp;
+	struct janus_pp_extension_report_rotation* next;
```

Comment:
This seems overkill. You can define `janus_pp_extension_report_rotation` without that and just rely on GLib's `GList` or `GSlist` for the job.